rln/src/public.rs

```diff
@@ -839,6 +838,30 @@ impl RLN<'_> {
         Ok(())
     }
 
+    // Generate RLN Proof using a witness calculated from outside zerokit
+    //
+    // output_data is  [ proof<128> | root<32> | epoch<32> | share_x<32> | share_y<32> | nullifier<32> | rln_identifier<32> ]
```

Comment:
```suggestion
    // output_data is  [ proof<128> | root<32> | external_nullifier<32> | x<32> | y<32> | nullifier<32>]
```
it is little bit different format of output data now regarding the structure

Reply:
indeed, good catch. fixed: 06393ae025da6c01257ac8c5aaf42d1ea2923583